tsdb/head.go

```diff
@@ -1282,9 +1382,19 @@ func (a *headAppender) Commit() (err error) {
 		return errors.Wrap(err, "write to WAL")
 	}
 
+	// No errors logging to WAL, so pass the exemplars along to the in memory storage.
+	for _, e := range a.exemplars {
+		s := a.head.series.getByID(e.ref)
+		if err := a.exemplarAppender.AddExemplar(s.lset, e.exemplar); err != nil {
+			// todo: do we want to error out here?
```

Comment:
Since this is experimental, let's not fail the commit on this error. Can we have a metric tracking these failures? Maybe also log an error here.